library/src/ui/add-col-btn/addColBtn.js

```diff
@@ -0,0 +1,17 @@
+window.CoNDeT.ui.AddColumnComponent = (function () {
+  function constructor() {}
+
+  constructor.prototype = Object.create(window.CoNDeT.ui.BaseComponent);
+  constructor.prototype.typeId = "AddColumnComponent";
+
+  constructor.prototype.createRef = function () {
+    return document.createElement("span");
+  };
+
+  constructor.prototype.onInit = function () {
+    this.ref.className = "add-col-element";
+    this.ref.innerHTML = "❎";
+  };
```

Comment:
Jak robimy spanem to pasuje dodać, aby był a11y. Ja przerobiłem twojego poprzedniego spana (delete table) aby był a11y. Podejrzyj tam i popraw analogicznie tutaj, ok?